.github/workflows/release.yml

```diff
@@ -96,7 +91,84 @@ jobs:
           release rollback --yes
   rm:
     runs-on: ubuntu-latest
-    if: inputs.job_to_run == 'rm' && inputs.tb_deploy
+    if: inputs.job_to_run == 'rm'
+
+    defaults:
+      run:
+        working-directory: ${{ inputs.data_project_dir }}
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+          architecture: "x64"
+          cache: 'pip'
+
+      - name: Validate input
+        run: |
+          [[ "${{ secrets.tb_admin_token }}" ]] || { echo "Go to the tokens section in your Workspace, copy the 'admin token' and set TB_ADMIN_TOKEN as a Secret in your Git repository"; exit 1; }
+      - name: Install Tinybird CLI
+        run: |
+          if [ -f "requirements.txt" ]; then
+            pip install -r requirements.txt
+          else
+            pip install tinybird-cli
+          fi
+      - name: Tinybird version
+        run: tb --version
+
+      - name: Remove Release
+        run: |
+          source .tinyenv
+          tb \
+          --host ${{ secrets.tb_host }} \
+          --token ${{ secrets.tb_admin_token }} \
+          release rm  \
+          --semver $VERSION --force --yes
+
+  dry_run_rm_oldest_rollback:
+    name: DRY-RUN Remove latest rollback Release
+    runs-on: ubuntu-latest
+    if: inputs.job_to_run == 'dry_run_rm_oldest_rollback'
+
+    defaults:
+      run:
+        working-directory: ${{ inputs.data_project_dir }}
+    steps:
+      - uses: actions/checkout@master
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.11"
+          architecture: "x64"
+          cache: 'pip'
+
+      - name: Validate input
+        run: |
+          [[ "${{ secrets.tb_admin_token }}" ]] || { echo "Go to the tokens section in your Workspace, copy the 'admin token' and set TB_ADMIN_TOKEN as a Secret in your Git repository"; exit 1; }
+      - name: Install Tinybird CLI
+        run: |
+          if [ -f "requirements.txt" ]; then
+            pip install -r requirements.txt
+          else
+            pip install tinybird-cli
+          fi
+      - name: Tinybird version
+        run: tb --version
+
+      - name: Remove Release
+        run: |
+          source .tinyenv
+          tb \
+          --host ${{ secrets.tb_host }} \
+          --token ${{ secrets.tb_admin_token }} \
+          release rm  \
+          --oldest-rollback --force --dry-run
+
+  rm_oldest_rollback:
+    name: DRY-RUN Remove latest rollback Release
```

Comment:
Looks like this one is the one without DRY-RUN, so we need to update this name.
Also the command sent in this job at the end still have the `--dry-run`, I guess in this case it should have `--yes`

Reply:
Good catch 🤦 Thanks!